### PR TITLE
SKY UX 3 migrate

### PR DESCRIPTION
**Pull request by Blackbaud-PaulCrowder**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@blackbaud/skyux-builder-plugin-auth-email-whitelist",
```
Comment (Blackbaud-BobbyEarl) [inline package.json:2]: Do we want to republish this under the `@skyux-sdk` scope?  Related question for @Blackbaud-SteveBrush, did we update the plugin logic to accept both of those, or is the intention to load plugins from the `@blackbaud`scope?

Comment (Blackbaud-PaulCrowder) [reply package.json:2]: We can, but it does work as-is.  I tested it on a SKY UX 3 SPA and it got pulled in as expected.

**Review by Blackbaud-BrandonJones — APPROVED** [review]

**Comment by Blackbaud-BrandonJones** [comment]
Thanks for doing this :) LGTM

**Review by Blackbaud-SteveBrush — APPROVED** [review]
LGTM.

**Pull request merged**